### PR TITLE
kvm: fix/optimize propogating configs

### PR DESCRIPTION
**Pull request by ustcweizhou**
## Description

Make some changes based on @nvazquez 's comments in PR #3491 
Fix a bug in #3491 

## Types of changes

- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] New feature (non-breaking change which adds functionality)
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] Enhancement (improves an existing feature and functionality)
- [ ] Cleanup (Code refactoring and cleanup, that may add test cases)

### CODE REVIEW

**Comment by rhtyd** [comment]
@blueorangutan package

**Comment by DaanHoogland** [comment]
@blueorangutan test

**Comment by DaanHoogland** [comment]
is this an implementation for #3896 , @weizhouapache ?

**Comment by rhtyd** [comment]
@nvazquez can you review? cc @DaanHoogland @andrijapanicsb

File: core/src/main/java/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java [hunk 1]
```diff
@@ -319,9 +319,12 @@ private Answer execute(GetDomRVersionCmd cmd) {
     }
 
     public boolean configureHostParams(final Map<String, String> params) {
-        if (_params.get("router.aggregation.command.each.timeout") == null) {
+        if (_params.get("router.aggregation.command.each.timeout") != null) {
             String value = (String)params.get("router.aggregation.command.each.timeout");
-            _eachTimeout = Duration.standardSeconds(NumbersUtil.parseInt(value, 10));
+            _eachTimeout = Duration.standardSeconds(NumbersUtil.parseLong(value, 600));
```
Comment (GabrielBrascher) [inline core/src/main/java/com/cloud/agent/resource/virtualnetwork/VirtualRoutingResource.java]: I am +1 on externalizing 600 into a constant.

**Comment by rhtyd** [comment]
I'm not sure what happened, I squashed merge this PR on 4.13 and the commit now says I authored it. What just happened cc @ustcweizhou @weizhouapache @DaanHoogland @andrijapanicsb

**Comment by DaanHoogland** [comment]
no idea, @rhtyd . time skew? you stole apache code?

**Comment by rhtyd** [comment]
I certainly did not steal the commit, if you see git history it clearly hints github did the squash+merge. I would blame bug in @github @DaanHoogland. I'll try and use rebase+merge in future where PRs have single commit.

**Comment by DaanHoogland** [comment]
i wouldn't worry to much @rhtyd . let's keep an eye out on it.

**Pull request merged**